### PR TITLE
Standardizing cheat sheet key press syntax

### PR DESCRIPTION
**Pull request by talsraviv**
https://app.asana.com/0/29464726597674/39381601893520/f

### CODE REVIEW

File: duckduckhack/goodie/goodie_cheat_sheets.md [hunk 1]
```diff
@@ -167,8 +166,15 @@ You might express consecutive key presses as follows:
 	
 #### Alternative Keys (e.g., pressing either A or B)
 
-You can express alternatives as follows:
+We recommend displaying alternatives as follows: 
+
+- As separate code blocks, separated by a forward slash, e.g. `"[?!=]/[?<!]"`
```
Comment (talsraviv) [inline duckduckhack/goodie/goodie_cheat_sheets.md]: @moollaza @abeyang what do you think about this case when the entire key combo has an alternative - no need for parens, right?

Comment (moollaza) [inline duckduckhack/goodie/goodie_cheat_sheets.md]: Russell had suggested that in these cases we should duplicate the key/value, and have the description indicate it's an alternate. 

That may be best,  especially when the shortcut is long

**Comment by talsraviv** [comment]
@moollaza check it out

**Comment by moollaza** [comment]
LGTM :+1:

**Pull request merged**